Templates/Integrations/AnalyticsAndSIEM/AnalyticsAndSIEM.py

```diff
@@ -0,0 +1,356 @@
+    description: str = args.get('description', '')
+    assignee: List[str] = argToList(args.get('assignee', ''))
+    raw_response = client.update_event_request(event_id, description=description, assignee=assignee)
+    event = raw_response.get('event')
+    if event:
+        title: str = f'{client.integration_name} - Event `{event_id}` has been updated.'
+        context_entry = build_context(event)
+        context = {f'{client.integration_context_name}.Event(val.ID && val.ID === obj.ID)': context_entry}
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        raise DemistoException(f'{client.integration_name} - Could not update event `{event_id}`')
+
+
+def create_event(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    description: str = args.get('description', '')
+    assignee: List[str] = argToList(demisto.args().get('assignee', ''))
+    raw_response: Dict = client.create_event_request(description, assignee)
+    event: Dict = raw_response.get('event', {})
+    if event:
+        event_id: str = event.get('eventId', '')
+        title = f'{client.integration_name} - Event `{event_id}` has been created.'
+        context_entry = build_context(event)
+        context = {f'{client.integration_context_name}.Event(val.ID && val.ID === obj.ID)': context_entry}
+        human_readable = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        raise DemistoException(f'{client.integration_name} - Could not create new event.')
+
+
+def query(client: Client, args: Dict):
+    query_dict = {
+        'eventId': args.get('event_id'),
+        'sinceTime': args.get('since_time'),
+        'assignee': argToList(args.get('assignee')),
+        'isActive': args.get('is_active') == 'true'
+    }
+    # Remove None/empty object
+    query_dict = {key: value for key, value in query_dict.items() if vault is not None}
+    if not query_dict.get('assignee'):
+        del query_dict['assignee']
+    raw_response: Dict = client.query_request(**query_dict)
+    events: List = raw_response.get('event', [])
+    if events:
+        title = f'{client.integration_name} - Results for given query'
+        context_entry = build_context(events)
+        context = {f'{client.integration_context_name}.Event(val.ID && val.ID === obj.ID)': context_entry}
+        human_readable: str = tableToMarkdown(title, context_entry)
+        return human_readable, context, raw_response
+    else:
+        return_warning(f'{client.integration_name} - Could not find any results for given query')
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def main():
+    """ GLOBALS/PARAMS """
+    integration_name = 'Analytics & SIEM Integration'
+    # lowercase with `-` dividers
+    integration_command_name = 'analytics-and-siem'
+    # No dividers
+    integration_context_name = 'AnalyticsAndSIEM'
+    server: str = demisto.params().get('url', '')
```

Comment:
first init `params: Dict = demisto.params()` and then get the params out of it
it is just faster